rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -959,6 +997,14 @@ public synchronized void registerLanguage(LanguageParameter lang) {
         multiplexer.addContributor(buildContributionKey(lang),
             new InterpretedLanguageContributions(lang, this, availableWorkspaceService(), (IBaseLanguageClient)clientCopy, exec));
 
+        try {
+            availableCapabilities().updateCapabilities(contributions).get();
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+        } catch (ExecutionException e) {
+            logger.error("Unexpected error while updating dynamic capabilities", e);
+        }
```

Comment:
This reads like we're joining on a future, why are we doing this?

Reply:
Ok, then we might need to revisit some other places where we use `get`. Anyway, removed this one!